<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>609</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1009</width>
    <height>609</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GeoAlgorithm</string>
  </property>
  <property name="windowOpacity">
   <double>0.950000000000000</double>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(41, 43, 47);</string>
  </property>
  <property name="iconSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1011</width>
      <height>611</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="button_start">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>60</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(91, 97, 106);
color: white;</string>
      </property>
      <property name="text">
       <string>Generuj</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_a">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="inputMask">
       <string>#99.9</string>
      </property>
      <property name="text">
       <string>-4.0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_d">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">;color: rgb(255, 255, 255);</string>
      </property>
      <property name="inputMask">
       <string>9</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>20</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>a</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>b</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_b">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="inputMask">
       <string>#99.9</string>
      </property>
      <property name="text">
       <string>12.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>20</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>d (10^-x)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_t">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(255, 255, 255);</string>
      </property>
      <property name="inputMask">
       <string>99999</string>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>20</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>T</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QTableWidget" name="best_table">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>601</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::SelectedClicked</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>2</number>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>200</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>200</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>name</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>value</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>real</string>
       </property>
       <property name="textAlignment">
        <set>AlignLeading|AlignVCenter</set>
       </property>
       <property name="background">
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </property>
      </column>
      <column>
       <property name="text">
        <string>bin</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>fx</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>Xreal</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>Xbin</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>fx</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
       <property name="background">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string/>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
     </widget>
     <widget class="QChartView" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>180</y>
        <width>1021</width>
        <height>401</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(41, 43, 47);</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>60</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>zatrzymaj na MAX (10.999)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>40</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>T</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_generations_test">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>40</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
color: rgb(255, 255, 255);</string>
      </property>
      <property name="inputMask">
       <string>99999</string>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_test_generations">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>70</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(91, 97, 106);
color: white;</string>
      </property>
      <property name="text">
       <string>Testuj pokolenia</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="test_table">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>401</width>
        <height>451</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
Header {Background-color: rgb(190,1,1);}</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::SelectedClicked</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>133</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string notr="true">tau</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
       <property name="background">
        <color>
         <red>41</red>
         <green>43</green>
         <blue>47</blue>
        </color>
       </property>
       <property name="foreground">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </property>
      </column>
      <column>
       <property name="text">
        <string>T</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
       <property name="background">
        <color>
         <red>41</red>
         <green>43</green>
         <blue>47</blue>
        </color>
       </property>
       <property name="foreground">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </property>
      </column>
      <column>
       <property name="text">
        <string>f(x)</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
       <property name="background">
        <color>
         <red>41</red>
         <green>43</green>
         <blue>47</blue>
        </color>
       </property>
       <property name="foreground">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>iteracje</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>wystÄ…pienia</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>%</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
     </widget>
     <widget class="QChartView" name="widget_test" native="true">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>120</y>
        <width>571</width>
        <height>451</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(41, 43, 47);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_a_test">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255)</string>
      </property>
      <property name="inputMask">
       <string>#99.9</string>
      </property>
      <property name="text">
       <string>-4.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>b</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>d (10^-x)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>21</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>a</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_b_test">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="inputMask">
       <string>#99.9</string>
      </property>
      <property name="text">
       <string>12.0</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="input_d_test">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">;color: rgb(255, 255, 255);</string>
      </property>
      <property name="inputMask">
       <string>9</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtChart</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
